nh_eobs/views/observation_report_template.xml

```diff
@@ -813,41 +818,10 @@
-                                                                <td class="col-xs-12">
-                                                                    <table class="compact-data col-xs-12">
-                                                                        <tr>
-                                                                            <td colspan="2" class="header col-xs-12">
-                                                                                <h4><t t-if="'summary' in task and task['summary']"><t t-esc="task['summary']"/></t></h4>
-                                                                            </td>
-                                                                        </tr>
-                                                                        <t t-if="'notes' in task and task['notes']">
-                                                                            <tr>
-                                                                                <td colspan="2" class="col-xs-12"><t t-esc="task['notes']"/></td>
-                                                                            </tr>
-                                                                        </t>
-                                                                        <tr>
-                                                                            <td class="col-xs-6"><t t-if="'user_id' in task and task['user_id']"><strong>By:</strong> <t t-esc="task['user_id'][1]"/></t></td>
-                                                                            <td class="col-xs-6"><t t-if="'date_terminated' in task and task['date_terminated']"><strong>Date:</strong> <t t-esc="task['date_terminated']"/></t><br/>
-                                                                                <t t-if="'date_terminated' in task and task['date_terminated']==False">
-                                                                                        Task still in progress
-                                                                                </t>
-                                                                            </td>
-                                                                        </tr>
-                                                                    </table>
-                                                                </td>
-                                                            </tr>
-                                                        </t>
-                                                    </table>
-                                                </td>
-                                            </tr>
+                                            <t t-call="nh_eobs.triggered_actions"/>
+                                        </t>
+                                        <t t-if="'state' in observation and observation['state'] == 'cancelled'">
```

Comment:
I looked at the template and thought that too, the diff is messing with my head though, I think it's actually fine! Check it out [here](https://github.com/NeovaHealth/openeobs/blob/fede6251edb19b121ccb8c49589072eac53866f0/nh_eobs/views/observation_report_template.xml#L819) instead.

I did a test where I stopped and restarted obs to cause a cancellation, and then did an obs with some other escalation tasks. The cancelled obs appears correctly in chronological order.